source/develop/release-management/features/storage/pass_discard_from_guest_to_underlying_storage.html.md

```diff
@@ -0,0 +1,234 @@
+- It should be supported by the guest's OS.
+
+#### The virtual machine's disk
+It is possible to enable *"Pass Discard"* for two disk storage types: **Images** and **Direct LUNs**.<br/>
+Regardless of the type, the disk interfaces that support *"Pass Discard"* are **VirtIO SCSI** and **IDE**.
+
+Note that for *Direct LUNs* with a *VirtIO SCSI* interface, enabling the *"Enable SCSI Pass-Through"* property already gives the disk *discard* support (among other things).<br/>
+We still need to decide about the relationship between these two fields - it should be discussed with the UX team (**<font color="red"> TBD </font>**).
+
+#### *Wipe After Delete* and *Pass Discard*
+
+##### Block storage
+When a disk's *Wipe After Delete* property is enabled, the engine guarantees that when the user removes the disk, it would be first wiped and only then removed.<br/>
+Since discarding a block in the storage means that it doesn't belong to the lv anymore, and since we cannot wipe this block before we discard it, if a disk's *Wipe After Delete* and *Pass Discard* properties were both enabled, we could have exposed some sensitive data as part of a new disk that was allocated with the same blocks to another user.<br/>
+Thus, *Wipe After Delete* and *Pass Discard* cannot, theoretically, be both enabled for the same disk.
+
+There is a case, though, where it's possible. To understand it, let's first take a look at these declarations:
+
+- Let *lunX* be a device and *dm-X* be its corresponding dm device for a natural number X. Then *lunX* supports the property that ***discard zeroes the data*** iff the value of */sys/block/dm-X/queue/discard_zeroes_data* is 1 [^2].
+A lun that supports the property that *discard zeroes the data* guarantees that previously discarded blocks are read back as zeros from it.
+- A storage domain supports the property that *discard zeroes the data* iff **all** of the luns that it is built from support it. That means that if at least one of its luns doesn't support the property that *discard zeroes the data*, then so is the storage domain.
+
+Thus, *Wipe After Delete* and *Pass Discard* can be both enabled for the same disk iff the corresponding storage domain supports the property that *discard zeroes the data*.
+
+##### File storage
+On file storage, the file system guarantees that previously removed blocks are read back as zeros. Thus, we never wipe removed disks on file storage like we do in block storage, because the file system does the work for us.<br/>
+Therefore, we always allow to enable both *Wipe After Delete* and *Pass Discard* for a disk on file storage, but we will **not** allow to move or copy this disk to a block domain that doesn't support *discard* as is, i.e the *Pass Discard* property of the new/copied disk will be set to false.
+
+
+## Usage
+
+### GUI
+> **Note**<br/>
+> The next screenshots were taken when the VM was down.<br/>
+> When the VM is up, *Pass Discard* **cannot** be edited.
```

Comment:
Worth checking if qemu/libvirt allow this, and if so, enabling it in some future version.